### DISCUSSION TITLE
How to map MITRE tags to NIDS rules

### DISCUSSION THREAD

**Discussion by IamTenacious** [Security-Onion-Solutions/securityonion#7384] [Unsupported Versions] [answered]
Hi, I wanna know how can I map MITRE tags to NIDS rules(snort, suricata and zeek)

**Comment by johnununu** [comment, marked as answer]
https://docs.securityonion.net/en/latest/managing-alerts.html?highlight=so-rule#so-rule

**Reply by johnununu** [reply]
You can also add modified rules directly to the manager sls file in '/opt/so/saltstack/local/pillar/minions/'
something like:

idstools:
  sids:
    disabled:
    enabled:
    modify:
    - alert_number "original_content" "new_content" '###remember to escape [ ", |, ! ] special chars with \'

**Reply by IamTenacious** [reply]
Thank you for the reply. the solution should be different for me because I have used the python script to convert the json events or attributes into the snort, suricata rules and the zeek script, in these generated rules I wanna add the MITRE tags which will kind of indicate the coverage so how can I do that?

**Reply by IamTenacious** [reply]
[https://github.com/SigmaHQ/sigma/blob/master/rules/windows/registry_event/registry_event_add_port_monitor.yml
](url)

look at how these tags are assigned to sigma rules (line 31 to 33) in similar way how can we assign mitre tags to the NIDS rules so that those tags can be used for further work.

**Reply by dougburks** [reply]
You may be able to add to the `metadata` section of the NIDS rule.

**Reply by IamTenacious** [reply]
Thank you.
I tried that but I am unable to see the MITRE tags in the 'HUNT' section of SO, where can I see that?

**Reply by weslambert** [reply]
If defined in a rule as a tag, the value will show up in `tags`. If defined in a rule as a key:value metadata pair, then it will be available in `rule.metadata.mitre*`

**Reply by IamTenacious** [reply]
Thank you so much!!!
can you also tell me how can I map MITRE tags in zeek's **.intel** file as well

**Reply by dougburks** [reply]
The Zeek Intel Framework format has several metadata fields:
https://docs.zeek.org/en/master/frameworks/intel.html

**Reply by IamTenacious** [reply]
Thank you!
